src/test/java/com/github/cameltooling/lsp/internal/util/RouteTextBuilder.java

```diff
@@ -38,4 +50,78 @@ public static String createXMLBlueprintRoute(String camelUri) {
 		return XML_PREFIX_FROM + camelUri + XML_SUFFIX_FROM_BLUEPRINT;
 	}
 
+	/**
+	 * @param javaClassContent
+	 * @return builds an empty Java class with the specified content and the cursor position inside and after
+	 *          contents
+	 */
+	public static DocumentContentWithPosition createJavaDocumentClass(String javaClassContent) {
+		String newLine = System.getProperty("line.separator");
```

Comment:
left this on as it makes us sure that our tests will work with windows and linux based line breaks. Tests run fine so it's good that it's working just fine :)